lib/gssapi/mech/gss_import_name.c

```diff
@@ -262,6 +266,8 @@ gss_import_name(OM_uint32 *minor_status,
 	HEIM_TAILQ_FOREACH(m, &_gss_mechs, gm_link) {
 		int present = 0;
 
+                _gss_mg_log(10, "attempting mech \"%s\"", m->gm_mech.gm_name);
```

Comment:
I like the `gdn` prefix you used in `gss_display_name()`.  Can you add a `gin` prefix here?